catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -0,0 +1,445 @@
+package ddf.catalog.history;
```

Comment:
Missing license.  Please run `mvn clean install` to find other checkstyle issues.